addons/velaux/resources/apiserver.cue

```diff
@@ -17,6 +19,10 @@ output: {
 			image: parameter["repo"] + "/" +"oamdev/vela-apiserver:" + parameter["version"]
 		}
 
+		if parameter["imagePullSecrets"] != _|_ {
+			imagePullSecrets: json.Unmarshal(parameter["imagePullSecrets"])
```

Comment:
No need to use JSON.

Reply:
As I explained in last comment, because the input parameters of vela cli is "string", I parse "string" to "array"